protostar/commands/call_command.py

```diff
@@ -0,0 +1,98 @@
+import json
+from logging import Logger
+from typing import Any, Optional, Dict
+
+from starknet_py.net.gateway_client import GatewayClient
+
+from protostar.cli import (
+    NetworkCommandUtil,
+    ProtostarArgument,
+    ProtostarCommand,
+)
+from protostar.starknet_gateway import GatewayFacadeFactory
+
+
+class CallCommand(ProtostarCommand):
+    def __init__(
+        self,
+        logger: Logger,
+        gateway_facade_factory: GatewayFacadeFactory,
+    ):
+        self._logger = logger
+        self._gateway_facade_factory = gateway_facade_factory
+
+    @property
+    def name(self) -> str:
+        return "call"
+
+    @property
+    def description(self) -> str:
+        return "Calls a contract on StarkNet with given parameters"
+
+    @property
+    def example(self) -> Optional[str]:
+        return None
+
+    @property
+    def arguments(self):
+        return [
+            *NetworkCommandUtil.network_arguments,
+            ProtostarArgument(
+                name="contract-address",
+                description="The address of the contract being called.",
+                type="str",
+                is_required=True,
+            ),
+            ProtostarArgument(
+                name="function",
+                description="The name of the function being called.",
+                type="str",
+                is_required=True,
+            ),
+            ProtostarArgument(
+                name="inputs",
+                description="Inputs to the function being called, represented by a list of space-delimited values.",
+                type="felt",
```

Comment:
```suggestion
                type="felt",
```
felt or int?

Reply:
inputs are of the same type as in `invoke`. so i guess yes - felt